api/Window.json

```diff
@@ -2199,7 +2199,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
```

Comment:
This is post-Presto. There's a bunch of that, but the PR isn't too big, so I'll just change the description instead.